WordPressKit/Decodable+Dictionary.swift

```diff
@@ -0,0 +1,100 @@
+struct JSONCodingKeys: CodingKey {
+    var stringValue: String
+
+    init?(stringValue: String) {
+        self.stringValue = stringValue
+    }
+
+    var intValue: Int?
+
+    init?(intValue: Int) {
+        self.init(stringValue: "\(intValue)")
+        self.intValue = intValue
+    }
+}
+
+
+/// Add support do decode to a Dictionary
+extension KeyedDecodingContainer {
+
+    func decode(_ type: Dictionary<String, Any>.Type, forKey key: K) throws -> Dictionary<String, Any> {
```

Comment:
Should we add unit tests for this extension?

Reply:
Yes, we should! df0da44bf24ed784990a9d1489c26a3eaeed77b9